.config/cargo_spellcheck.dic

```diff
@@ -35,6 +35,9 @@ scalable
 adjunctive
 invariants
 deserializes
+Dapp
+untyped
+subber
```

Comment:
Not sure we should include words and names that do not really exist.
Maybe we find a better way here?

Reply:
I've added a solution for `untyped` and `subber`, but would leave `DApp` in there, since imho it's an established term.

Reply:
`untyped` and `DApp` are fine imo.